src/visual/Visualizer.js

```diff
@@ -86,6 +86,20 @@ export default class Visualizer {
     this._timer = null;
     this._step = null;
     this.clear();
+
+    this.paperDefaultValidateConnectionMethod = this.paper.options.validateConnection;
+
+    this.paper.options.validateConnection = (cellViewS, magnetS, cellViewT, magnetT, end, linkView) => {
+      const args = [cellViewS, magnetS, cellViewT, magnetT, end, linkView];
+
+      if (this.paperDefaultValidateConnectionMethod.call(this.paper, ...args)) {
```

Comment:
This is very strong notice and I followed this to make code seems more cleanly. Thank you very much.

Done in 3b68ddd37a commit